README.md

```diff
@@ -36,49 +36,25 @@ Run the application:
 $ oneview-redfish-toolkit
 ```
 
- At first time, it will create all the needed configuration files under user directory, and will prompt for the OneView IP(s).
+At first time, it will create all the needed configuration files under user directory, and will prompt for the OneView IP(s) to customize `redfish.conf` file with the entered IP. The following files will be created:
 
-You can update the configuration files created under the user directory, or if you want to use custom configuration files you can pass them as arguments:
+* $HOME/.config/oneview-redfish-toolkit/
+  * redfish.conf
+  * logging.conf
+  * redfish.log
+  * redfish_performance.log
+  * redfish_ov_data.log
+  * certs/
 
-```bash
-$ oneview-redfish-toolkit --config redfish.conf --log-config logging.conf
-```
+The `redfish.conf` and `logging.conf` are used for toolkit customization and certs directory is used to place SCMB certificates retrieved from OneView instances when Event Service is enabled.
```

Comment:
The `redfish.conf` and `logging.conf` are used for toolkit customization, but generally don't need to be modified.  The certs directory is used to place the retrieved OneView certificates when the Event Service is enabled.

Reply:
Done.